src/Test/TestCommanderAve.java

```diff
@@ -120,6 +121,9 @@ boolean testSnowSpread()
       }
     }
     //System.out.println("Snow map:\n" + Ave.getSnowMapAsString());
+    Unit infantry = addUnit(testMap, Ave, UnitModel.TROOP, snowCity.xCoord, snowCity.yCoord);
+    GamePath path = Utils.findShortestPath(infantry, new XYCoord(10, 5), testMap);
+    testPassed &= validate( 2 == path.getFuelCost(Ave, infantry.model, testMap), "    Ave's units burn too much fuel moving through snow." );
```

Comment:
This seems unrelated to the rest of the test it's in, but I guess some test is better than no test.